lib/veri_report.mli

```diff
@@ -0,0 +1,24 @@
+open Core_kernel.Std
+open Bap.Std
+open Bap_traces.Std
+open Regular.Std
+
+type event = Trace.event
+
+type t [@@deriving bin_io, sexp]
+
+type events = Value.Set.t
+
+type frame_diff = {
```

Comment:
field names are in the `Veri_report` namespace and it is kind of hard to get an access to them. It would be better to put it int the `Frame_diff` module and expose field functions. Or maybe it is even better just to use a pair for a frame diff. Since our fields are already names as `left` `right`, switching to a pair (that has natural left and right parts) will not loose to much semantics.